I don't know why, but this got reverted upstream by the original commit author.

But it works fine for us, as we even patched this in before they did it, and
I've emailed them.
diff --git a/Makefile.am b/Makefile.am
new file mode 100644
index 0000000..63d9994
--- /dev/null
+++ b/Makefile.am
@@ -0,0 +1,3 @@
+info_TEXINFOS = c.texi
+c_TEXINFOS = cpp.texi fp.texi fdl.texi gpl.texi
+dist_data_DATA = c.pdf
diff --git a/configure.ac b/configure.ac
new file mode 100644
index 0000000..911616b
--- /dev/null
+++ b/configure.ac
@@ -0,0 +1,11 @@
+# Process this file with autoconf to produce a configure script:
+# autoreconf -vi
+# Public domain. Originally written by Ineiev, 2023.
+AC_INIT([GNU C Intro And Reference], [0.0-rc3], [c-manual@gnu.org],
+  [c-intro-and-ref], [https://www.gnu.org/software/c-intro-and-ref/])
+
+AC_PREREQ([2.65])
+AC_CONFIG_SRCDIR([c.texi])
+AM_INIT_AUTOMAKE([1.11])
+AC_CONFIG_FILES([Makefile])
+AC_OUTPUT

